common/types.go

```diff
@@ -474,8 +474,14 @@ type WebhookVerificationParameters struct {
 	Headers      http.Header
```

Comment:
can you expand on the comment? Is the WebhookVerificationParameters initialized per webhook we receive?